Resources/Prototypes/_Goobstation/Loadouts/loadout_groups.yml

```
@@ -72,9 +74,20 @@
   loadouts:
   - BlueshieldOfficerOuterBlueshieldMantle
 
+# Reporter
+
 - type: loadoutGroup
   id: ReporterOuterClothing
   name: loadout-group-reporter-outerclothing
   loadouts:
   - NoPro
 
+#sec
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)